tests/tasks/networking/test_netmiko_run.py

```diff
@@ -4,9 +4,10 @@
 class Test(object):
 
     def test_netmiko_run(self, brigade):
+        cmd_kwargs = {'command_string': 'hostname'}
         result = brigade.filter(name="dev4.group_2").run(networking.netmiko_run,
                                                          method="send_command",
-                                                         command_string="hostname")
+                                                         cmd_kwargs=cmd_kwargs)
```

Comment:
To reiterate from the last comment. I think what's in the current tests for this and the current implementation is much cleaner.

Reply:
Yeah, I agree, I did that change while writing the tests and trying to figure out some weird error that said something like `e` is not a valid something (I send the command without being enclosed in an iterable). I don't have much experience with netmiko so my error was due to not knowing the library but we should assume users using this task don't know the library either. Otherwise they could just do `host.get_connection('netmiko').send_command(...)` themselves :P

Reply:
As per above, I will probably fix this in `netmiko_send_command` as the connection arguments are going away so it doesn't make sense to do it there.

It still does make sense to use the original solution with netmiko_run, however, as netmiko_run can call any Netmiko method and there are a lot of them. 

In other words, since any Netmiko method can be called, it makes sense to have clear separation between the arguments that are for the task and the arguments that will be passed to the method. It will also make it clearer on the documentation. It will also make it less likely to introduce end-user code problems due to changes in the task definition.

Reply:
Note that users can always do:

```
def something(task):
    task.host.get_connection(“netmiko”).netmiko_method(arg1, arg2)

brigade.run(something)
```

So basically our tasks have to provide value over the simplicity of just those 2 lines.